debugger/src/main/kotlin/org/rust/debugger/settings/RsDebuggerSettings.kt

```diff
@@ -23,6 +23,7 @@ class RsDebuggerSettings : XDebuggerSettings<RsDebuggerSettings>("Rust") {
     var downloadAutomatically: Boolean = false
 
     var breakOnPanic: Boolean = true
+    var stepSettings: RsStepFilterSettings = RsStepFilterSettings(filterStdlib = true)
```

Comment:
I think we should leave the default behaviour as is
```suggestion
    var stepSettings: RsStepFilterSettings = RsStepFilterSettings(filterStdlib = false)
```